guide/popular-topics/embeds.md

```diff
@@ -224,25 +223,25 @@ We will now explain how to edit embedded message content and resend a received e
 
 ### Resending a received embed
 
-To forward a received embed you retrieve it from the messages embed array (`message.embeds`) and pass it to the MessageEmbed can then be edited before sending it again.
+To forward a received embed you retrieve it from the messages embed array (`message.embeds`) and pass it to the EmbedBuilder, then it can be edited before sending it again.
 
 ::: warning
 We deliberately create a new Embed here instead of just modifying `message.embeds[0]` directly to keep the cache valid. If we were not to do this, the embed in cache on the original message would diverge from what the actual embed looks like, which can result in unexpected behavior down the line!
```

Comment:
`message.embeds[0]` isn't a builder anymore so you can't modify it, might want to rephrase that sentence a bit